src/utils/queue.ts

```diff
@@ -0,0 +1,45 @@
+import 'source-map-support/register'
```

Comment:
Apparently there are arguments against including this in libraries, cause it doesn't play nicely with webpacking things